templates/vue/src/components/Combobox.vue

```diff
@@ -151,6 +152,11 @@ export default {
     getValue(option) {
       return typeof option === "string" ? option : option[this.itemValue]
     },
+    decodeHtmlEntity(str) {
+      return str.replace(/&#(\d+);/g, function(match, dec) {
+        return String.fromCharCode(dec)
+      })
+    },
```

Comment:
Seeing as you're calling this method regularly, and on change to the input, this might be better done as a computed property.

Reply:
Ended up not needing this method, so it was removed.